### DISCUSSION TITLE
metrics.lecoq.io got 526 from cloudflare.

### DISCUSSION THREAD

**Discussion by JasonkayZK** [lowlighter/metrics#1275] [Q&A] [answered]
The perhaps the certificate has been expired, and all generated resources can not work recently:

![image](https://user-images.githubusercontent.com/40811521/199403901-5403341b-bf14-4f76-9ad0-cf44d9f57f51.png)

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

Thanks a lot for reporting it

The cert was renewed but the nginx server wasn't reloaded so it was still using the old one
This has been fixed now 👍

**Reply by JasonkayZK** [reply]
Thanks for fixing this, you guys are so cool! ❤️
